conf/application.conf

```diff
@@ -153,6 +163,11 @@ Prod {
         port = 80
       }
 
```

Comment:
This will never be 'localhost' for any non-local environment. Perhaps safest to put the prod url as a default? At least that way it works without further configuration in production. The same point applies to the address-lookup config too.